store/sqlstore/channel_member_history_store.go

```diff
@@ -109,47 +115,42 @@ func (s SqlChannelMemberHistoryStore) hasDataAtOrBefore(time int64) (bool, error
 }
 
 func (s SqlChannelMemberHistoryStore) getFromChannelMemberHistoryTable(startTime int64, endTime int64, channelId string) ([]*model.ChannelMemberHistoryResult, error) {
-	query := `
-			SELECT
-				cmh.*,
-				u.Email,
-				u.Username,
-				Bots.UserId IS NOT NULL AS IsBot,
-				u.DeleteAt AS UserDeleteAt
-			FROM ChannelMemberHistory cmh
-			INNER JOIN Users u ON cmh.UserId = u.Id
-			LEFT JOIN Bots ON Bots.UserId = u.Id
-			WHERE cmh.ChannelId = :ChannelId
-			AND cmh.JoinTime <= :EndTime
-			AND (cmh.LeaveTime IS NULL OR cmh.LeaveTime >= :StartTime)
-			ORDER BY cmh.JoinTime ASC`
-
-	params := map[string]interface{}{"ChannelId": channelId, "StartTime": startTime, "EndTime": endTime}
+	query, args, err := s.getQueryBuilder().
+		Select("cmh.*, u.Email, u.Username, Bots.UserId IS NOT NULL AS IsBot, u.DeleteAt AS UserDeleteAt").
+		From("ChannelMemberHistory cmh").
+		Join("Users u ON cmh.UserId = u.Id").
+		LeftJoin("Bots ON Bots.UserId = u.Id").
+		Where(sq.Eq{"cmh.ChannelId": channelId}).
+		Where(sq.LtOrEq{"cmh.JoinTime": endTime}).
+		Where(sq.Or{
+			sq.Eq{"cmh.LeaveTime": nil},
+			sq.GtOrEq{"cmh.LeaveTime": startTime},
+		}).
+		OrderBy("cmh.JoinTime ASC").ToSql()
+	if err != nil {
+		return nil, err
+	}
 	var histories []*model.ChannelMemberHistoryResult
-	if _, err := s.GetReplica().Select(&histories, query, params); err != nil {
+	if _, err := s.GetReplica().Select(&histories, query, args...); err != nil {
 		return nil, err
 	}
 
 	return histories, nil
 }
 
 func (s SqlChannelMemberHistoryStore) getFromChannelMembersTable(startTime int64, endTime int64, channelId string) ([]*model.ChannelMemberHistoryResult, error) {
-	query := `
-		SELECT DISTINCT
-			ch.ChannelId,
-			ch.UserId,
-			u.Email,
-			u.Username,
-		    Bots.UserId IS NOT NULL AS IsBot,
-			u.DeleteAt AS UserDeleteAt
-		FROM ChannelMembers AS ch
-		INNER JOIN Users AS u ON ch.UserId = u.id
-		LEFT JOIN Bots ON Bots.UserId = u.Id
-		WHERE ch.ChannelId = :ChannelId`
-
-	params := map[string]interface{}{"ChannelId": channelId}
+	query, args, err := s.getQueryBuilder().Select("ch.ChannelId, ch.UserId, u.Email, u.Username, Bots.UserId IS NOT NULL AS IsBot, u.DeleteAt AS UserDeleteAt").
```

Comment:
```suggestion
	query, args, err := s.getQueryBuilder().
	Select("ch.ChannelId, ch.UserId, u.Email, u.Username, Bots.UserId IS NOT NULL AS IsBot, u.DeleteAt AS UserDeleteAt").
```
nit could the select be on a new line? Thanks